app/controllers/photos_controller.rb

```diff
@@ -1,5 +1,10 @@
 class PhotosController < ApplicationController
   def index
     @photos = Photo.all
+    if Cart.last
+    @cart = Cart.last.total_unique_items
```

Comment:
Also, I'm a little bit worried about setting `@cart` with `Cart.last` what guarantee do you have here that the last `Cart` object in the database belongs to the current user? 🤔